From a727027af285699ef85ab52caac3c629d657cc7e Mon Sep 17 00:00:00 2001
From: William Douglas <william.douglas@intel.com>
Date: Mon, 23 Oct 2023 10:56:35 -0700
Subject: [PATCH 10/12] Disable bootstrap by default

We do not want to enable this feature yet, so make sure it is disabled
out of the box.

Signed-off-by: Patrick McCarty <patrick.mccarty@intel.com>
Signed-off-by: William Douglas <william.douglas@intel.com>
---
 docs/site-defaults.cfg | 5 +++--
 py/mock.py             | 2 +-
 py/mockbuild/config.py | 4 ++--
 3 files changed, 6 insertions(+), 5 deletions(-)

diff --git a/docs/site-defaults.cfg b/docs/site-defaults.cfg
index f8bd9dd..9a58f92 100644
--- a/docs/site-defaults.cfg
+++ b/docs/site-defaults.cfg
@@ -129,7 +129,8 @@
 # Therefore, with `use_bootstrap=True` default, we first install a minimal
 # "bootstrap" chroot that only contains rpm/yum/dnf stack, and from that
 # "bootstrap" chroot we initialize the target chroot.
-#config_opts['use_bootstrap'] = True
+# NOTE: In Clear Linux OS, the default value is False.
+#config_opts['use_bootstrap'] = False
 
 #config_opts['system_yum_command'] = '/usr/bin/yum'
 #config_opts['system_dnf_command'] = '/usr/bin/dnf'
@@ -140,7 +141,7 @@
 # case, you may need to have "bootstrap" chroot pre-populated from a container
 # image first (where the package manager stack is already pre-installed, so
 # mock doesn't have to).
-#config_opts['use_bootstrap_image'] = True
+#config_opts['use_bootstrap_image'] = False
 #config_opts['bootstrap_image'] = 'fedora:latest'
 
 # Mock in a nutshell needs to have the selected config_opts["package_manager"]
diff --git a/py/mock.py b/py/mock.py
index 19b517b..e2e571c 100755
--- a/py/mock.py
+++ b/py/mock.py
@@ -715,7 +715,7 @@ def main():
 
     # When scrubbing all, we also want to scrub a bootstrap chroot
     if options.scrub:
-        config_opts['use_bootstrap'] = True
+        config_opts['use_bootstrap'] = False
 
     # outer buildroot to bootstrap the installation - based on main config with some differences
     bootstrap_buildroot = None
diff --git a/py/mockbuild/config.py b/py/mockbuild/config.py
index eb2f483..0c3c8b9 100644
--- a/py/mockbuild/config.py
+++ b/py/mockbuild/config.py
@@ -77,8 +77,8 @@ def setup_default_config_opts():
     config_opts['nspawn_args'] = ['--capability=cap_ipc_lock']
     config_opts['use_container_host_hostname'] = True
 
-    config_opts['use_bootstrap'] = True
-    config_opts['use_bootstrap_image'] = True
+    config_opts['use_bootstrap'] = False
+    config_opts['use_bootstrap_image'] = False
     config_opts['bootstrap_image'] = 'fedora:latest'
     config_opts['bootstrap_image_ready'] = False
     config_opts['bootstrap_image_fallback'] = True
-- 
2.42.0

